share.md

```diff
@@ -13,8 +13,8 @@ ev3dev.org Projects Page
 We currently have a [projects page] where you can browse projects that have been
 built using ev3dev. You can add your own too!
 
-{% assign project = site.categories.projects | first %}
-Here is the most recent project ([more about this project...]({{ site.github.url }}{{ project.url}})).
+{% assign project = site.categories.projects | last %}
+Here is the sample project ([more about this project...]({{ site.github.url }}{{ project.url}})).
```

Comment:
I think we should also have a [link](https://raw.githubusercontent.com/ev3dev/ev3dev.github.io/master/projects/_posts/2014-03-21-Example-Project.md) to the "source" of the sample project on GitHub so that it can easily be copied.